Lib/fontTools/misc/psCharStrings.py

```diff
@@ -410,17 +413,28 @@ def op_index(self, index):
 	def op_roll(self, index):
 		raise NotImplementedError
 
-	# TODO(behdad): move to T2OutlineExtractor and add a 'setVariation'
-	# method that takes VarStoreData and a location
 	def op_blend(self, index):
 		if self.numRegions == 0:
 			self.numRegions = self.private.getNumRegions()
 		numBlends = self.pop()
 		numOps = numBlends * (self.numRegions + 1)
-		del self.operandStack[-(numOps-numBlends):] # Leave the default operands on the stack.
+		if self.blender is None:
+			del self.operandStack[-(numOps-numBlends):] # Leave the default operands on the stack.
+		else:
+			argi = len(self.operandStack) - numOps
+			end_args = tuplei = argi + numBlends
+			while argi < end_args:
+				next_ti = tuplei + self.numRegions
+				deltas = self.operandStack[tuplei:next_ti]
+				delta = self.blender(self.vsIndex, deltas)
+				self.operandStack[argi] += otRound(delta)
```

Comment:
Is rounding here kosher? Don't we get an accumulation of rounding errors this way? Same at the HVAR site.

Reply:
To answer myself: no, it is not kosher. I have a test case that shows it:
<img width="1064" alt="image" src="https://user-images.githubusercontent.com/4246121/187030795-e62aa189-a312-4cb6-b4de-23be81303ab9.png">
This is the cap A from MutatorSans compiled as CFF2. Those points at the top align properly without the round (as they do in the TTF VF).